is_core/forms/forms.py

```diff
@@ -123,7 +123,7 @@ def _clean_fields(self):
 
     def _register_readonly_field(self, field_name):
         if field_name not in self.readonly_fields:
-            self.readonly_fields.append(field_name)
+            self.readonly_fields.add(field_name)
```

Comment:
`self.readonly_fields` is set, see above:
https://github.com/matllubos/django-is-core/pull/390/files#diff-bacc6244284719575eb07a9150230c7cR61

Reply:
outh thx